### ISSUE TITLE
TuiGAN: Learning Versatile Image-to-Image Translation with Two Unpaired Images

### ISSUE THREAD

**Issue by IsHYuhi** [IsHYuhi/PaperSummary#8]
# INFO
## author
Jianxin Lin<sup>1*</sup>, Yingxue Pang<sup>1*</sup>, Yingce Xia<sup>2</sup>, Xhibo Chen<sup>1†</sup>, and Jiebo Luo<sup>3</sup>
## affiliation
<sup>１</sup> CAS Key Laboratory of Technology in Geo-spatial Information Processing and Application System, University of Science and Technology of China, <sup>2</sup> Microsoft Research Asia, <sup>3</sup> University of Rochester
## conference or year
ECCV 2020
## link
[arXiv](https://arxiv.org/abs/2004.04634)
[実装](https://github.com/linjx-ustc1106/TuiGAN-PyTorch)

# 概要
![image](https://user-images.githubusercontent.com/38097069/98648002-0792ac80-2379-11eb-9f4c-40fb0c7e3a21.png)
* 2枚のunpairedな画像のみで学習するTuiGANを提案.
* cGANをピラミッド構造に並べ, 画像を粗いレベルから詳細レベルに段階的に翻訳する.
* 十分な量のデータで学習する既存のunsupervised image-to-image translation(UI2I) SoTAモデルと同等の性能を達成.　様々なUI2Iタスクに対応できることを実証した.

# 提案手法
既存の ones-shot cross-domain変換問題において逆方向の変換は難しかった. 提案手法では2枚のunpairedなイメージにおいて両方向の変換を実現することに着目をした.
SinGANにおいて, ピラミッド構造のGANをpatch distributionを異なるスケールにおいて学習させている. しかし1枚の画像のみしか考慮していないのでペア間の関係を捉えてはいない.  対照的に提案手法においてソースとドメインの分布の関係を学習し, 構造的を保ったまま変換を行うことを可能にした.
### network architecture
![image](https://user-images.githubusercontent.com/38097069/98912754-db5a6580-2509-11eb-88de-1a5125fad0cf.png)
![image](https://user-images.githubusercontent.com/38097069/98914264-c979c200-250b-11eb-8dd0-b2e5a0ae18db.png)
![image](https://user-images.githubusercontent.com/38097069/98913443-b61a2700-250a-11eb-81eb-aa9a46d21f5a.png)

データが非常に限られているので, マルチスケール(scale factor (1/s)<sup>n</sup>)なGANを用いて段階的に画像を洗練していく. (低解像度->高解像度, ↑はbicubicによるupsampling)

Attention maskとpixel-wiseに掛けることで, generatorが現在のスケールの詳細部と, 一つ前のスケールの大域的な構造をうまく保つように合成する.

# Loss Functions
* Adversarial Loss(WGAN-GP)
* cycle-consistency loss
* identity loss
![image](https://user-images.githubusercontent.com/38097069/98914085-96373300-250b-11eb-8e08-4c1dfe07cfdc.png)
* total variation loss

# 検証
### 定性的評価
![image](https://user-images.githubusercontent.com/38097069/98916267-4c038100-250e-11eb-9807-58f39bac471a.png)

### 定量的評価
![image](https://user-images.githubusercontent.com/38097069/98916964-22972500-250f-11eb-90be-775b1eed5c14.png)

### ablation study
![image](https://user-images.githubusercontent.com/38097069/98916982-2c208d00-250f-11eb-8318-9d4136918a52.png)
![image](https://user-images.githubusercontent.com/38097069/98919352-08ab1180-2512-11eb-9294-842cb586cbab.png)

# 新規性
* 2枚のunpairedな画像のみで学習するTuiGANを提案.
* 様々なUI2Iタスクに対応できることを実証した.
# 議論，展望
# Comment
同じ1ペアによる画像変換↓
[Structural-analogy from a Single Image Pair](https://arxiv.org/abs/2004.02222)
# date
November 12th, 2020
